release.py

```diff
@@ -721,7 +721,7 @@ def select_gpg_key():
 Release notes for the %(release_version)s release:
 https://home.apache.org/~%(apache_id)s/kafka-%(rc_tag)s/RELEASE_NOTES.html
 
-*** Please download, test and vote by <VOTING DEADLINE, e.g. Monday, March 28, 9am PT>
+*** Please download, test and vote by <VOTING DEADLINE, e.g. Monday, March 28, 9am PT, which should be at least 72 hours from now.>
```

Comment:
can you please add the reason for 72 hours here as well? You can provide a link to https://www.apache.org/legal/release-policy.html#release-approval.

Reply:
Thank you for reviewing the change @divijvaidya. I added a little more details as suggested. Please let me know if not sufficient.